lineid_plot.py

```diff
@@ -418,10 +426,11 @@ def plot_line_ids(wave, flux, line_wave, line_label1, label1_size=None,
     # Redraw the boxes at their new x location.
     for i in range(nlines):
         box = ax.texts[i]
-        if hasattr(box, 'xyann'):
```

Comment:
No, unfortunately, I made this change as a hack >6 months ago and forgot to submit the PR.  

I think ditching the "if" statement entirely and just doing a "hasattr" check is good enough.  Change forthcoming.